wiki/resources/official-badges.md

```diff
@@ -328,7 +328,7 @@ Discord badges are epic-looking achievements to make your profile look cool. The
       <td>Only for the Clyde Ai bot.</td>
    </tr>
    <tr>
-      <td>Ai</td>
+      <td>Original Poster</td>
```

Comment:
Lets change that to "OP (Original Poster)" to stay consisten with the original repo!

Reply:
Hey @SJ-Tom would you mind fixing this?

Reply:
Hey @mezotv is this better now?

Reply:
Yea thats better thanks!